### ISSUE TITLE
How to plot region as in Fig 3E in Lovell et al eLife 11:e78526

### ISSUE THREAD

**Issue by matt-shenton** [jtlovell/GENESPACE#99]
HI there,

Thanks for the great tool.

I may be being foolish, but I'm struggling to find how to make a sub-chromosomal region riparian plot like Fig3E in the paper. Or was that done outside the package?

Many thanks for your help

Matt Shenton

**Comment by jtlovell** [comment]
That is done with the riparian plotting function ... see section 5.2 of the tutorial [here](https://htmlpreview.github.io/?https://github.com/jtlovell/tutorials/blob/main/riparianGuide.html)

**Comment by matt-shenton** [comment]
Thanks for your response. My problem was that I didn't realise I need to specify all the genomes in the roi. I only specified the position in the reference genome, then the whole chromosome was displayed for all the genomes. Now I realized that if I specify all genomes with their appropriate syntenic regions, then I can get the plot I wanted. Thanks for your help, and sorry for bothering you.
Best wishes
Matt Shenton
